bridges/AnidexBridge.php

```diff
@@ -131,8 +133,16 @@ public function collectData() {
 			$opt[CURLOPT_COOKIE] = 'anidex_h_toggle=' . $h;
 		}
 
+		// We need to use a different Host HTTP header to reach the correct page on ALTERNATE_URI
+		$headers = array('Host: ' . self::ALTERNATE_HOST);
+
+		// The HTTPS certificate presented by anidex.moe is for anidex.info. We need to ignore this.
```

Comment:
AKA [Domain Fronting](https://en.wikipedia.org/wiki/Domain_fronting). Never saw this being used in real life...

Thanks for these comments.

Reply:
You're welcome. 😉